webpack.config.dist.js

```diff
@@ -0,0 +1,23 @@
+require("babel/register");
+
+const defaultConfig = require( './webpack.config.js' ),
+	webpack = require( 'webpack' ),
+	WebpackStrip = require( 'strip-loader' );
+
+module.exports = Object.assign( {}, defaultConfig, {
+	output: {
+		path: __dirname + '/dist',
+		filename: 'wpcom.js',
+		libraryTarget: 'umd',
+		library: 'WPCOM'
+	},
+	loaders: [
+		{
+			test: /\.js$/,
+			loader: WebpackStrip.loader('debug', 'console.log', 'console.warn')
```

Comment:
Not sure if we want to try doing something like this for the standalone distribution, but it's a cool feature we can take advantage of and stop bugging users in the DevTools console.